phpstan.neon.dist

```diff
@@ -7,3 +7,11 @@ parameters:
     level: max
     paths:
         - src
+    treatPhpDocTypesAsCertain: false
+    exceptions:
+        check:
+            missingCheckedExceptionInThrows: true
+            tooWideThrowType: true
+        uncheckedExceptionClasses:
+            - 'RuntimeException'
+            - 'BadMethodCallException'
```

Comment:
It is specific for this code:
```
protected function createStream(): StreamInterface
    {
        throw new \BadMethodCallException('Not implemented');
    }
```

I believe it is not goal to force developer to wrap this exception with try/catch